udata/core/badges/models.py

```diff
@@ -14,15 +17,9 @@
 
 
 class Badge(db.EmbeddedDocument):
-    kind = db.StringField(choices=[], required=True)
+    kind = db.StringField(required=True)
     created = db.DateTimeField(default=datetime.now, required=True)
     created_by = db.ReferenceField('User')
-    removed = db.DateTimeField()
-    removed_by = db.ReferenceField('User')
-
-    meta = {
-        'ordering': ['created'],
```

Comment:
A default ordering can be useful to memorize the consistent place of the badge in a list for mass addition for instance. To be confirmed by admins :)